project/Dependencies.scala

```diff
@@ -24,7 +24,7 @@ object Dependencies {
   val java8CompatV = "0.9.0"
   val jgitV = "5.13.0.202109080827-r"
   val metaconfigFor211V = "0.9.10" // metaconfig stops publishing for scala 2.11
```

Comment:
Do we need to publish scalafix for `2.11` though? Isn't metaconfig only needed for CLI/sbt ?

Reply:
Scalafix must be published in 2.11 for [`ExplicitResultTypes`](https://github.com/scalacenter/scalafix/issues/998) to work against 2.11 sources. This is something we can challenge though.

metaconfig is included in community rules projects via `scalafix-core` and is [documented](https://scalacenter.github.io/scalafix/docs/developers/tutorial.html#use-withconfiguration-to-make-a-rule-configurable) 
```
$ cs resolve -t com.github.liancheng::organize-imports:0.6.0
└─ com.github.liancheng:organize-imports_2.13:0.6.0
   ├─ ch.epfl.scala:scalafix-core_2.13:0.9.31
   │  ├─ com.geirsson:metaconfig-typesafe-config_2.13:0.9.14
   │  │  ├─ com.geirsson:metaconfig-core_2.13:0.9.14
```